descriptors/ModuleDescriptor-template.json

```diff
@@ -174,6 +179,20 @@
           "pathPattern": "/_/tenant"
         }
       ]
+    },
+    {
+      "id": "_timer",
+      "version": "1.0",
+      "interfaceType": "system",
+      "handlers": [
+        {
+          "methods": [ "POST" ],
+          "pathPattern": "/loadHoldings",
+          "modulePermissions": ["configuration.entries.collection.get"],
+          "unit": "day",
+          "delay": "2"
+        }
```

Comment:
Didn't realize timer functionality is available -- do we have any way to control when (for example time of day) this periodic task will happen

Reply:
@carolegodfrey we are not able to specify the exact time, the time of the run will count from the module enabled for the tenant. For instance, module enabled at 24 May 2019 09:21:54 then the first run of the scheduled task will be 26 May 2019 09:21:54

Reply:
as for the future improvement we can do following - after the first time of setting RM API configuration we can do a call to load holdings, no matter when the module was enabled.

Reply:
Thanks for clarification @natali-zaitseva